client/asset/dcr/dcr.go

```diff
@@ -6198,7 +6588,7 @@ func (dcr *ExchangeWallet) handleTipChange(ctx context.Context, newTipHash *chai
 		dcr.cycleMixer()
 	}
 
-	go dcr.checkPendingTxs(ctx, uint64(newTipHeight))
+	go dcr.syncTxHistory(ctx, uint64(newTipHeight))
```

Comment:
We now are launching two unmonitored goroutines in `handleTipChange`. Maybe we should make `handleTipChange` synchronous and use a waitgroup internally.